botfront/imports/api/endpoints/endpoints.methods.js

```diff
@@ -6,7 +6,9 @@ import { checkIfCan } from '../../lib/scopes';
 
 export const createEndpoints = async (project) => {
     if (!Meteor.isServer) throw Meteor.Error(401, 'Not Authorized');
-    if (!process.env.ORCHESTRATOR) return;
+    if (!process.env.ORCHESTRATOR) {
+        process.env.ORCHESTRATOR = 'docker-compose';
```

Comment:
You shouldn't set values on process.env, it's not the intended purpose.
use an intermediate variable instead like
`const orchestration = process.env.ORCHESTRATOR ? process.env.ORCHESTRATOR : 'docker-compose';`